docs/mindsdb-docs/docs/contribute/integrations.md

```diff
@@ -35,39 +34,39 @@ integrations                      # Contains integrations source codes
 
 In technical terms, a handler is a self-contained Python package having everything required for MindsDB to interact with it. It includes aspects like dependencies, unit tests, and continuous integration logic. It is up to the author to determine the nature of the package, for example, closed or open source, version control, and more. Although, we encourage opening pull requests to expand the default set of supported tools.
 
-The entry point is a class definition that should inherit either from the `integrations.libs.base.DatabaseHandler` class or the `integrations.libs.base.PredictiveHandler` class, depending on the type of the handler. The `integrations.libs.base.BaseHandler` class defines all the methods that must be overwritten in order to achieve a functional implementation.
+The entry point for a database handler is a class definition that should inherit directly from the `integrations.libs.base.DatabaseHandler` class and thus indirectly from the `integrations.libs.base.BaseHandler` class, which defines all the methods that must be overwritten in order to achieve a functional implementation.
+
+And the entry point for an ML handler is a class definition that should inherit from the `integrations.libs.base.BaseMLEngine` class, which defines all the methods that must be overwritten in order to achieve a functional implementation.
```

Comment:
Just for further clarification, I would append to this: 

> Note that the mapping from a `BaseMLEngine` to a `BaseHandler` is automatically performed by an internal wrapper, so that writing an integration to an ML framework is as easy as possible.